mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -367,12 +367,12 @@ func resourceMongoDBAtlasClusterCreate(d *schema.ResourceData, meta interface{})
 	projectID := d.Get("project_id").(string)
 	providerName := d.Get("provider_name").(string)
 
-	autoScaling := matlas.AutoScaling{
+	autoScaling := &matlas.AutoScaling{
```

Comment:
looks like some of the other fixes as well based on Gustavo's feedback?   Can you note on your PR what all to expect and why?

Reply:
As Edgar incorporated the new attributes in the client in order to be used in terraform, he needed to update the client's dependency, what it brings all the new changes made, responded to the first question.

Reply:
Okay so sorta related but not quite.   I think for tracking it would make help to note what was done, especially for any new team members, for example just add to the PR like the bolded area:

Added new fields **and tests** for data sources and resource of snapshot restore jobs:
    oplogTs
    oplogInc
    pointInTimeUTCSeconds
**Change attributes as required by newest go client version**